prov/sm2/src/sm2.h

```diff
@@ -328,29 +334,15 @@ int sm2_cntr_open(struct fid_domain *domain, struct fi_cntr_attr *attr,
 
 int64_t sm2_verify_peer(struct sm2_ep *ep, fi_addr_t fi_addr);
 
-void sm2_format_pend_resp(struct sm2_tx_entry *pend, struct sm2_cmd *cmd, void *context,
-			  enum fi_hmem_iface iface, uint64_t device,
-			  const struct iovec *iov, uint32_t iov_count, uint64_t op_flags,
-			  int64_t id, struct sm2_resp *resp);
-void sm2_generic_format(struct sm2_cmd *cmd, int64_t peer_id, uint32_t op, uint64_t tag,
-			uint64_t data, uint64_t op_flags);
-size_t sm2_copy_to_sar(struct smr_freestack *sar_pool, struct sm2_resp *resp,
-		       struct sm2_cmd *cmd, enum fi_hmem_iface, uint64_t device,
-		       const struct iovec *iov, size_t count, size_t *bytes_done,
-		       int *next);
-size_t sm2_copy_from_sar(struct smr_freestack *sar_pool, struct sm2_resp *resp,
-			 struct sm2_cmd *cmd, enum fi_hmem_iface iface, uint64_t device,
-			 const struct iovec *iov, size_t count, size_t *bytes_done,
-			 int *next);
-
-int sm2_select_proto(bool use_ipc, bool cma_avail, enum fi_hmem_iface iface, uint32_t op,
-		     uint64_t total_len, uint64_t op_flags);
+void sm2_generic_format(struct sm2_free_queue_entry *cmd, int64_t peer_id, uint32_t op,
+			uint64_t tag, uint64_t data, uint64_t op_flags);
+int sm2_select_proto();
```

Comment:
I'm guessing this does nothing now - you can just remove it completely now that we're not assuming parallel development with shm

Reply:
I kept it in there b/c I have more patches lined up to add additional features.  I can remove it for now.

Reply:
removed